chaosoci/compute/common.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, List, Dict
+
+from oci.core import ComputeClient
+
+from chaoslib.exceptions import ActivityFailed
+
+__all__ = ["get_instances", "filter_instances"]
+
+
+def get_instances(client: ComputeClient = None,
+                  compartment_id: str = None) -> List:
```

Comment:
Returns a List of ?

Reply:
Would this impact something I am not aware of? or just in terms of improving annotations?

Reply:
The latter. It's more documentation and helping editors.

Reply:
We may also enable mypy someday.